### DISCUSSION TITLE
'useVirtualList' was not found

### DISCUSSION THREAD

**Discussion by Aaz21rus** [vueuse/vueuse#2316] [Q&A]
Hello!
I've got such WARNING:
"export 'useVirtualList' was not found in '@vueuse/core'

My package.json :
"vue": "^2.6.11",
"@vueuse/core": "^5.1.3",
"@vue/composition-api": "1.0.5",

Maybe I have old version?

**Comment by qcp** [comment]
Yep, it's exist since [v6.4.0](https://github.com/vueuse/vueuse/releases/tag/v6.4.0)
